packages/event-bridge/src/events/index.ts

```diff
@@ -32,19 +45,9 @@ export type PocketEvent =
   | ExportReady
   | ExportRequested
   | PremiumPurchaseEvent
-  | AddItem
-  | DeleteItem
-  | FavoriteItem
-  | UnfavoriteItem
-  | ArchiveItem
-  | UnarchiveItem
-  | AddTags
-  | ReplaceTags
-  | ClearTags
-  | RemoveTags
-  | RenameTag
-  | DeleteTag
-  | UpdateTitle;
+  | ListEvent
```

Comment:
I realized moving each "root type" to a Union type, makes it easier to use and be specific in other services.